SpaceCore/Patches/GameLocationPatcher.cs

```diff
@@ -35,10 +35,10 @@ public override void Apply(HarmonyInstance harmony, IMonitor monitor)
                 postfix: this.GetHarmonyMethod(nameof(After_Explode))
             );
 
-            harmony.Patch(
-                original: this.RequireMethod<GameLocation>(nameof(GameLocation.updateEvenIfFarmerIsntHere)),
-                postfix: this.GetHarmonyMethod(nameof(After_UpdateEvenIfFarmerIsntHere))
-            );
+            //harmony.Patch(
+            //    original: this.RequireMethod<GameLocation>(nameof(GameLocation.updateEvenIfFarmerIsntHere)),
+            //    postfix: this.GetHarmonyMethod(nameof(After_UpdateEvenIfFarmerIsntHere))
+            //);
```

Comment:
Because `After_UpdateEvenIfFarmerIsntHere` is taken out, naturally this patch will no longer work. So it's also taken out.